features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```diff
@@ -78,6 +78,10 @@ repos:
       sub_dir: cpp/libcudf_kafka
       depends: [cudf]
   python:
+    - name: pylibcudf
+      sub_dir: python/pylibcudf
+      depends: [cudf]
+      args: {cmake: -DFIND_CUDF_CPP=ON, install: *rapids_build_backend_args}
     - name: cudf
       sub_dir: python/cudf
       depends: [cudf]
```

Comment:
You'll want to change this to also depend on pylibcudf.

Reply:
https://github.com/rapidsai/devcontainers/pull/375#discussion_r1697682716

Reply:
Yeah, I think this doesn't require any changes.

I believe build-cudf does the blocks in order on the python side.
(i.e. cudf-kafka does not depend on cudf-python here even though it needs to compile against the cudf-python Cython)

Reply:
Oh you're totally right, I forgot how this gets used. This is just to set up the CMake arguments for build dependencies, Python builds just rely on the ordering of the manifest.